### PR TITLE
KAFKA-16921: migrate connect module to junit 5 (Part 2)

### PR DESCRIPTION
**Pull request by m1a2st**
https://issues.apache.org/jira/browse/KAFKA-16921
- org.apache.kafka.connect.integration

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectIntegrationTestUtils.java [hunk 1]
```diff
@@ -16,28 +16,26 @@
...
  */
-public class ConnectIntegrationTestUtils {
-    public static TestRule newTestWatcher(Logger log) {
-        return new TestWatcher() {
-            @Override
-            protected void starting(Description description) {
-                super.starting(description);
-                log.info("Starting test {}", description.getMethodName());
-            }
+public class ConnectIntegrationTestUtils implements TestWatcher, BeforeEachCallback, AfterEachCallback {
```
Comment (chia7712) [inline connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectIntegrationTestUtils.java]: It seems this is used to log test name. Maybe we can move the log to each tests? There are only three class using this helper.

**Review by chia7712 — COMMENTED** [review]
@m1a2st thanks for this patch

File: connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectIntegrationTestUtils.java (hunk 1 above)
Comment (m1a2st) [reply connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectIntegrationTestUtils.java]: It's a good idea, I will remove it

**Comment by m1a2st** [comment]
@chia7712, Thanks for your comment, PTAL

**Comment by chia7712** [comment]
@m1a2st could you please take a look at failed tests?

**Comment by chia7712** [comment]
failed tests pass on my local

**Pull request merged**